cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -149,6 +172,17 @@
             {% endif %}
         {% endfor %}
 
+        {% if is_share_view and is_printable %}
+            <li class="m-list_item m-social-media_print">
+                <button class="a-btn a-btn__link m-social-media_icon"
+                   onclick="{{ print_info.share_url }}"
+                   aria-label="{{ print_info.name }}"
+                   type="button">
```

Comment:
Pretttty sure this is superfluous.

```suggestion
                   aria-label="{{ print_info.name }}">
```